src/main/java/com/beowulfe/hap/impl/services/AbstractServiceImpl.java

```diff
@@ -39,13 +27,14 @@ public AbstractServiceImpl(String type) {
 	 *
 	 * @param type unique UUID of the service. This information can be obtained from HomeKit Accessory Simulator.
 	 * @param accessory HomeKit accessory exposed as a service.
+	 * @param serviceName name of the service. This information is usually the name of the accessory.
      */
-	public AbstractServiceImpl(String type, HomekitAccessory accessory) {
```

Comment:
Which, granted, wouldn't be called in any of the framework provided services - what you did with the constructors there makes sense. But still, people implementing their own services would likely find it convenient.